velox/common/serialization/DeserializationRegistry.cpp

```diff
@@ -17,15 +17,20 @@
 
 namespace facebook::velox {
 
-DeserializationRegistryType& DeserializationRegistryForSharedPtr() {
+DeserializationRegistryType& deserializationRegistryForSharedPtr() {
```

Comment:
This change breaks Prestissimo and needs to be (1) in a separate PR; (2) implemented in backwards compatible way.